src/components/WorldMap.js

```diff
@@ -6,6 +6,13 @@ import css from 'csz';
 import format from 'date-fns/format';
 import addDays from 'date-fns/addDays';
 import CountriesSearcher from './CountriesSearcher.js';
+import constats from '../services/servicesConfiguration';
+
+// ? Wrappers
```

Comment:
Is this comment really necessary? :eyes:

Reply:
We don't have code documentation so when I add for first time somtehing I left some comments